riff-raff.yaml

```diff
@@ -9,3 +9,10 @@ deployments:
       functionNames: [mobile-content-notifications-lambda-liveblogs-, mobile-content-notifications-lambda-]
       fileName: mobile-content-notifications-lambda.jar
       prefixStack: false
+   mobile-content-notifications-lambda-cfn:
```

Comment:
I seem to remember this wasn't working as the `riffraff` user in the CAPI account doesn't give us the right to do some of the operations. Has that changed?